swagger/v1/openapi.json

```diff
@@ -1078,6 +1078,160 @@
         }
       }
     },
+    "/value_definitions": {
```

Comment:
@marleystipich2 this needs to be generated by `bin/rake rswag:specs:swaggerize`. If the script is ran now, it removes the endpoint.

Also, the changes are not adding attributes that the changes are adding to the response. 

e.g:
```diff
                             "profiles": {
                               "data": [
                                 {
-                                  "id": "51b02806-b0d2-450d-9b7d-9cac0912e5cb",
+                                  "id": "1c947bc5-01f2-43bf-8e3e-d5af5eb144a0",
                                   "type": "profile"
                                 }
                               ]
+                            },
+                            "value_definitions": {
+                              "data": []
                             }
                           }
                         }
```

Reply:
The OpenAPI spec is generated from the contents of `spec/integration/` specs.

Reply:
I added the `value_definitions_spec` file and used that to generate the content in `openapi.json`. When you get a chance, can you check to see if it looks correct now?